TeslaHW/TeslaHW/Charging/ChargingView.swift

```diff
@@ -0,0 +1,324 @@
+            .fill(
+                Color.blue
+            )
+            .opacity(0.1)
+            .blur(radius: 8)
+            .frame(width: 150, height: 60)
+            .offset(x: -59, y: -30)
+    }
+    
+    private var indicateMiddleLayerView: some View {
+        ChargeProgressShape()
+            .fill(
+                LinearGradient(
+                    colors: [Color("GradientBottom"), Color("GradientTop")],
+                    startPoint: .bottom,
+                    endPoint: .top
+                )
+            )
+            .frame(width: 250, height: 39)
+            .scaleEffect(x: 0.6, anchor: .leading)
+            .padding(.trailing, 20)
+    }
+    
+    private var indicateBottomLayerView: some View {
+        ChargeProgressShape()
+            .fill(
+                LinearGradient(
+                    colors: [
+                        .gray.opacity(0.8),
+                        .gray.opacity(0.1),
+                        .black.opacity(0.1),
+                        .black.opacity(0.1),
+                        .black.opacity(0.4)
+                    ],
+                    startPoint: .top,
+                    endPoint: .bottom
+                )
+            )
+            .frame(width: 274, height: 39)
+            .scaleEffect(x: 1, anchor: .leading)
+    }
+    
+    private var persentLabelView: some View {
+        VStack {
+            HStack {
+                RoundedRectangle(cornerRadius: 2)
+                    .fill(Color.blue)
+                    .frame(width: 1, height: 10)
+                    .padding(.trailing, 60)
+                RoundedRectangle(cornerRadius: 2)
+                    .fill(Color.red.opacity(0.3))
+                    .frame(width: 1, height: 10)
+            }
+            .offset(x: 103, y: -20)
+            HStack {
+                Text(Constants.persentValue.0)
+                    .font(.footnote)
+                    .foregroundColor(.white)
+                    .padding(.trailing, 25)
+                Text(Constants.persentValue.1)
+                    .font(.footnote)
+                    .foregroundColor(.gray)
+            }
+            .offset(x: 90, y: -25)
+        }
+    }
+    
+    private var setChargeLimitView: some View {
+        VStack {
+            RoundedRectangle(cornerRadius: 3)
+                .fill(LinearGradient(colors: [
+                    .black,
+                    .black,
+                    .black,
+                    .gray.opacity(0.1),
+                    .white.opacity(0.1)
+                ], startPoint: .top, endPoint: .bottom))
+                .frame(width: 270, height: 8)
+                .overlay {
+                    Image(Constants.sliderImageName)
+                        .offset(x: 65, y: 5)
+                }
+            Text(Constants.limitText)
+                .font(.caption2)
+                .foregroundColor(.gray.opacity(0.4))
+        }
+    }
+    
+    private var nearbySuperchargesView: some View {
```

Comment:
```suggestion
    private var nearbySuperchargesView: some View {
```
Великоват, разнеси...